prod/mach_zehnder_gadget/javascript/app/WebSocketController.js

```diff
@@ -290,6 +421,80 @@ myApp.controller('WebSocketController',['$scope', function($scope) {
             results="";
             dps.splice(0,dps.length);
             xVal=0;
+            //Log Activity
+            var logObject = {
+                "objectType": "button",
+                "displayName":"sendExperimentResults"
+            }
+            actionLogger.logSend(logObject);
+        
+
+            var example_content = {
+               "concepts": [
+                  {
+                    "x": 297,
+                    "y": 188,
+                    "content": "energy",
+                    "id": "7f800d79-cd66-2167-724c-6c1cda7abc5e",
+                    "type": "ut_tools_conceptmapper_conceptSelector",
+                    "colorClass": "ut_tools_conceptmapper_blue"
+                  },
+                  {
+                    "x": 652,
+                    "y": 238,
+                    "content": "thermodynamic temperature",
+                    "id": "a1ad6ace-c722-ffa9-f58e-b4169acdb4e3",
+                    "type": "ut_tools_conceptmapper_conceptSelector",
+                    "colorClass": "ut_tools_conceptmapper_blue"
+                  }
+               ],
+               "relations": [
+                 {
+                    "source": "7f800d79-cd66-2167-724c-6c1cda7abc5e",
+                    "target": "a1ad6ace-c722-ffa9-f58e-b4169acdb4e3",
+                    "id": "con_71",
+                    "content": "influences"
+                 }
+               ]
+            };
+
+        var example_metadata = {
+          "published": "2014-11-12T09:27:14.186Z",
+          "storageType": "vault",
+          "storageId": "5458b041427dd6ce7cbf88675",
+          "actor": {
+            "objectType": "person",
+            "id": "57a3939b8bf26f18c08e5e12",
+            "displayName": "hagop"
+          },
+          "target": {
+            "objectType": "lab experiment",
+            "id": "unknown",
+            "displayName": "Mach Zehnder Experiment"
+          },
+          "generator": {
+            "objectType": "application",
+            "url": "http://128.178.96.66/GadgetTest/WebClients/gadget.xml#appId=57b1aa566afd990f54b07795&owner=57b1aa276afd990f54b0770d",
+            "id": null,
+            "displayName": "Mach Zehnder Interferometer"
+          },
+          "provider": {
+            "objectType": "ils",
+            "url": "http://graasp.eu/spaces/57b1aa276afd990f54b076fc",
+            "id": "57b1aa276afd990f54b076fc",
+            "displayName": "Test",
+            "inquiryPhase": "Investigation",
+            "inquiryPhaseId": "57b1aa276afd990f54b0770d",
+            "inquiryPhaseName": "Investigation"
+          }
+        };
+
+        ils.createResource("test", example_content, example_metadata,function(resource){
+          // write your code here to use the resource
+          // this line simply prints the created resource's JSON representation
+          console.log(resource);
+        });
+
     };
 
 }]);
```

Comment:
@hagopj13 plz take care of the styling issues and refer to the link for the coding convention to go through the whole code. Once done, plz squash and let me know